packages/model-viewer/src/features/scene-graph/image.ts

```diff
@@ -87,4 +110,12 @@ export class Image extends ThreeDOMElement implements ImageInterface {
     }
     this[$onUpdate]();
   }
+
+  applyTexture(applicator: ((texture: ThreeTexture) => void)|undefined) {
```

Comment:
Can we make this private? Meaning naming it with a symbol.

Reply:
Unfortunately I need to call it from Texture-Info, the overall architecture here is fighting against keeping this internal and believe me I tried. There is no concept of a friend class in typescript but I should be able to mimic it and thus hide this method, but it will just add more code. IMO with a little re-design we can have our cake and eat it too, but that's outside of this CL.